File: perl/src/tests/t_version.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$version = Atrshmlog::get_version();

$minorversion = Atrshmlog::get_minor_version();

$patchversion = Atrshmlog::get_patch_version();

print "version : $version : \n";
print "version : $minorversion : \n";
print "version : $patchversion : \n";

print " \n";

exit (0);

# end of main

File: python/src/tests/t_version.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

version = atrshmlog.get_version()

minorversion = atrshmlog.get_minor_version()

patchversion = atrshmlog.get_patch_version()

print('version : ' + str(version) + ' : ')
print('version : ' + str(minorversion) + ' : ')
print('version : ' + str(patchversion) + ' : ')


print (' ')

exit(0)

# end of test

